share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -136,6 +138,7 @@ default:
     - uname -a || true
     - grep -E 'vendor|model name' /proc/cpuinfo 2>/dev/null | sort -u || head -n10 /proc/cpuinfo 2>/dev/null || true
     - nproc || true
+    - python ${SPACK_ROOT}/share/spack/gitlab/cloud_pipelines/scripts/server/dynamic-mapping-server.py
```

Comment:
I'm kind of assuming the inclusion of a webserver which is started for every generation job is only here for testing purposes?

Reply:
correct. There are additional limitations on runners if this method is used, but I really just wanted to have a functioning MVP to test.

We could resolve those additional restrictions with environment variable expansion, but that is outside the scope of this feature.

Reply:
I removed all of the testing bits.